www/stats_operations.py

```diff
@@ -1,34 +1,48 @@
 import csv
 import time
+from datetime import datetime
 from .db import Stats
 from io import StringIO
 from flask import abort
 
 def read_stats():
     csv_output = StringIO()
 
-    writer = csv.DictWriter(csv_output, Stats._meta.fields.keys())
+    keys = Stats._meta.fields.keys();
+
+    new_keys = list(keys)
+    new_keys.remove('id')
+
+    writer = csv.DictWriter(csv_output, new_keys)
     writer.writeheader()
+    results = Stats.select(Stats.ref_id, Stats.project_name, Stats.osm_type, Stats.osm_id, Stats.timestamp, Stats.already_existed, Stats.action, Stats.user).dicts()
```

Comment:
> ## Bessere Lösung
> Allerdings bietet uns `peewee` mit `Stats._meta.fields` sowieso schon ein `dict`, das Feld-Namen auf die entsprechenden `Field`-Objekte mappt. Dieses könnten wir kopieren, den Eintrag für `'id'` entfernen, dann die keys für `DictWriter` verwenden und die values für `Stats.select`.

Das würde dann ungefähr so gehen:
```suggestion
    csv_fields = Stats._meta.fields.copy();
    csv_fields.pop('id')

    writer = csv.DictWriter(csv_output, csv_fields.keys())
    writer.writeheader()
    results = Stats.select(*csv_fields.values()).dicts()
```

Reply:
Ich habe dieser Version schlussendlich umgesetzt: hier in -> 63225933ac274a8a20cdc6230068a21dbb7014d9